duckduckgo/iOS
# added new full bleed icon and removed some unused icons (ios 5,6)

Task/Issue URL: https://app.asana.com/0/392891325557410/1132968255465407
Tech Design URL:
CC:

**Description**:

Update icon to use new full bleed icon.  Deleted some unused icons (iOS 5,6)

**Steps to test this PR**:
1. Install app, check new icon appears.
1. Check widgets
1. Check app switching
1. Check on iPhone/iPad


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
